src/views/Resources.vue

```diff
@@ -73,7 +73,7 @@ export default class Resources extends Vue {
   // match resourceinstance and corresponding type and search filter
   public resourceInstanceForType(resourceType: ResourceType): ListEntry[] {
     const resourcesPerType = this.resourceInstances.filter((resourceInstance) => {
-      return (resourceInstance.resourceType === resourceType.id &&
+      return (resourceInstance.resourceType.id === resourceType.id &&
```

Comment:
id somehow doesnt work here anymore, but you can compare the names to fix the problem.

Reply:
What do you mean this doesn't work? Are you using the latest backend version?

Reply:
I used Backend Commit 7321025fd4c77183cd98d83b4201811bee18e33a, which is 
"Merge pull request #26 from bptlab/feature/25-populate-parent-type". not the newsest, but close to it.
what i meant with "it doesnt work" is: it breaks the filter. the instances are not displayed anymore because it always returns false. 
you can check if there are any instances visible at your frontend with the newest backend and if so, I will re-check it on Thursday.

Reply:
Yeah with the newest backend version everything works perfect over here :)

Reply:
you are right, the newest backend does fix this.